<launch>

  <group ns="image_process_pkg">
  
    <node pkg="rqt_image_view" type="rqt_image_view" name="edge_detect" args="/image_edges"/>
    
    <node pkg="rqt_image_view" type="rqt_image_view" name="image_lines_white" args="/image_lines_white"/>
    
    <node pkg="rqt_image_view" type="rqt_image_view" name="image_lines_yellow" args="/image_lines_yellow"/>
    
    <node pkg="rqt_image_view" type="rqt_image_view" name="image_lines_all" args="/image_lines_all"/>
  
    <node pkg="image_process_pkg" name="hw7node" type="hw7node.py"/>
    
    <node pkg="image_process_pkg" name="hw8node" type="hw8node.py"/>
  
  </group>
  
  <node name="image_publisher" pkg="image_processing_hw" type="image_pub_all.py"/>
  
</launch>
  
  
    
